Manage measurement family (#528)
akeneo/pim-helpcenter: content/md/first-steps/manage-your-attributes.md

# How to create an attribute?

Akeneo proposes several types of attributes. Discover all attributes available in the [Introduction to Akeneo attribute types](what-is-an-attribute.html#definition-of-an-attribute) article. Make sure you pick the **most appropriate attribute type that perfectly matches your needs**.    

> For example, if you want to make sure that your PIM users will select one option from a list of predefined options, select a simple select attribute type instead of a free text field. Create a `Measurement` attribute to enter in the weight/size/length... of your products.

Reviewer: ```suggestion
For example, if you want to make sure that your PIM users will select one option from a list of predefined options, select a simple select attribute type instead of a free text field. Create a `Measurement` attribute to add a weight/size/length... for your products.
```
Sorry I replaced input with enter but didn't adapt the rest of the sentence...

Author: Thx
